tests/x11regressions/gnomecase/nautilus_permission.pm

```diff
@@ -14,13 +14,20 @@
 use base "x11regressiontest";
 use strict;
 use testapi;
+use version_utils 'sle_version_at_least';
 
 
 sub run {
     x11_start_program('touch newfile', valid => 0);
     x11_start_program('nautilus');
     send_key_until_needlematch 'nautilus-newfile-matched', 'right', 15;
-    send_key "shift-f10";
+    if (sle_version_at_least('15')) {
+        assert_and_click 'nautilus-newfile-matched', 'right';
+        record_soft_failure 'boo#1074057 qemu can not properly capture some keys in nautilus under GNOME wayland';
+    }
```

Comment:
Does it affect Leap15 or Tumbleweeed? and `is_sle` is needed in 'if' condition

Reply:
Fixed: http://10.67.17.30/tests/81